_includes/macros/sub-pages.html

```diff
@@ -0,0 +1,92 @@
+{# ==========================================================================
+
+   Sub-Pages
+
+   ==========================================================================
+
+   Description:
+
+   Builds sub-pages markup when given:
+
+   sub_pages      : An array containing sub-pags.
+
+   ========================================================================== #}
+
+{% macro render( sub_pages, show_in_office=false ) %}
+    {% set is_office_page = request.path.find('/offices/') > -1 %}
+    {% for page in sub_pages | sort(attribute='order') if is_office_page == false or
+                                                          page.show_in_office == "on" %}
+        {% if loop.first %}
+            <section class="office_sub_pages
+                          block
+                          block__sub
+                          block__padded-top
+                          block__border-top
+                          {{ "block__border-bottom" if is_office_page == false }}">
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/65720b50137ae7a4f8555a0a5bf1f019f5109f1b